concepts/05 Widgets/Scheduler/020 Data Binding/20 OData Service.md

```diff
@@ -91,6 +155,74 @@ Data kept in the **ODataStore** can be processed in the [DataSource](/api-refere
         [dataSource]="appointmentDataSource">
     </dx-scheduler>
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="dataSource"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxScheduler from 'devextreme-vue/scheduler';
+    import DataSource from 'devextreme/data/data_source';
+
+    const data = new DataSource({
+        store: {
+            type: "odata",
+            url: "http://url/to/odata/service"
+        },
+        filter: ["Not_Assigned", "=", true],
+        paginate: false
+    });
+
+    export default {
+        components: {
+            DxScheduler
+        },
+        data() {
+            return {
+                dataSource: data
```

Comment:
Why not just call the data source `dataSource`? This is the named used in the markup anyway.